packages/riipen-ui/src/components/Input.jsx

```diff
@@ -62,11 +60,9 @@ const Input = ({
     <div className={className}>
       {(label || hint) && (
         <InputLabel
-          color={labelColor}
           hint={hint}
           htmlFor={other.id || other.name}
           required={required}
-          weight={labelWeight}
         >
           {label}
         </InputLabel>
```

Comment:
One thing I'm a bit iffy on here is passing a custom InputLabel through via props.  This current code would look like `<InputLabel><InputLabel>{ .....} </InputLabel></InputLabel> `😬

Reply:
I think for right now, we leave `label` as a `string` only, not node. That would remove this concern. I think to truely switch this to a string or node, we need a slightly bigger PR which involves moving `<Typography>` out of the `InputLabel` component. Just not a change we need to do right now.

Reply:
Okay sounds good.  I'll close this PR!